### DISCUSSION TITLE
Perl brew home page's Perl version

### DISCUSSION THREAD

**Discussion by yuki-kimoto** [gugod/App-perlbrew#724] [Ideas]
Hi

Perl brew home page's Perl version is perl-5.16.0

https://perlbrew.pl/

I think this is a little old.

Can this be changed into Perl latest version?

**Comment by gugod** [comment]
https://github.com/gugod/App-perlbrew/wiki/index/_compare/b98c2d1b0b534728187bb47f32fd34f0e48a1c77...6de121aba7dac0565e4141ec471946fbfa25c64b

for the record this was done by edting the wiki page named `index`.
